### ISSUE TITLE
no inbound connections

### ISSUE THREAD

**Issue by mondbaron** [Chia-Network/chia-blockchain#6649]
I'm having the issue of no inbound connections since the first official release in my Debian 10 container. With the latest release I did a fresh install of chia-blockchain in a clean Ubuntu container but the node ***still does not establish inbound connections***. Exept from this everything works fine. Full Node and Wallet are synced and my Farmer responds to challenges.

Versions:
- running chia-blockchain 1.1.7
- in Python 3.8.5 virtual environment
- on Ubuntu 20.04

Install method to reproduce:
```
python3 -m venv venv
source venv/bin/activate
python -m pip install --upgrade pip
python -m pip install chia-blockchain
chia init
chia keys add
chia start farmer
```

Additionally, following the recommendations on this topic, I did disable the UPNP option in the config.

GOOD DETAILS:
- I did forward inbound TCP connections on port 8444 from my router to the chia machine
- I do see the incoming packets on port 8444 in the chia machine's firewall every few minutes
- by running `lsof -i TCP` I see chia processes listening on *:8444 (IPv4 and IPv6)

BAD DETAILS: 
- `chia show -c` does never show more than 8 connections, even after monitoring for a few days
- by running `lsof -i TCP` I could verify that all established connections on port 8444 are outbound only

Incoming traffic log looks like:
```
SRC=<someRemoteIp> DST=<myLocalIp> LEN=52 TOS=0x00 PREC=0x00 TTL=115 ID=39307 DF PROTO=TCP SPT=20531 DPT=8444 SEQ=3575687305 ACK=0 WINDOW=64240 SYN
```

`lsof -i TCP | grep 8444` produces
```
chia_full  1465 chia   25u  IPv6 60901679      0t0  TCP *:8444 (LISTEN)
chia_full  1465 chia   26u  IPv4 60901680      0t0  TCP *:8444 (LISTEN)
chia_full  1465 chia   30u  IPv4 60901694      0t0  TCP localhost:8444->localhost:48062 (ESTABLISHED)
chia_full  1465 chia   31u  IPv4 60902124      0t0  TCP <myLocalHostname>:33104-><someRemoteIp>:8444 (ESTABLISHED)
chia_full  1465 chia   33u  IPv4 60946040      0t0  TCP <myLocalHostname>:36896-><someRemoteIp>:8444 (ESTABLISHED)
chia_full  1465 chia   34u  IPv4 61182545      0t0  TCP <myLocalHostname>:43328-><someRemoteIp>:8444 (SYN_SENT)
chia_full  1465 chia   35u  IPv4 61089643      0t0  TCP <myLocalHostname>:57678-><someRemoteIp>:8444 (ESTABLISHED)
chia_full  1465 chia   36u  IPv4 60901707      0t0  TCP localhost:8444->localhost:48078 (ESTABLISHED)
chia_full  1465 chia   37u  IPv4 61082767      0t0  TCP <myLocalHostname>:52064-><someRemoteIp>:8444 (ESTABLISHED)
chia_full  1465 chia   38u  IPv4 61100939      0t0  TCP <myLocalHostname>:58388-><someRemoteIp>:8444 (ESTABLISHED)
chia_full  1465 chia   39u  IPv4 60947627      0t0  TCP <myLocalHostname>:56124-><someRemoteIp>:8444 (ESTABLISHED)
chia_full  1465 chia   40u  IPv4 61192613      0t0  TCP <myLocalHostname>:41642-><someRemoteIp>:8444 (SYN_SENT)
chia_full  1465 chia   42u  IPv4 61038550      0t0  TCP <myLocalHostname>:59558-><someRemoteIp>:8444 (ESTABLISHED)
chia_full  1465 chia   43u  IPv4 61184759      0t0  TCP <myLocalHostname>:57122-><someRemoteIp>:8444 (SYN_SENT)
chia_full  1465 chia   44u  IPv4 61184370      0t0  TCP <myLocalHostname>:55308-><someRemoteIp>:8444 (SYN_SENT)
chia_full  1465 chia   73u  IPv4 60906821      0t0  TCP <myLocalHostname>:45744-><someRemoteIp>:8444 (ESTABLISHED)
chia_wall  1468 chia   19u  IPv4 60905332      0t0  TCP localhost:48078->localhost:8444 (ESTABLISHED)
chia_full  1687 chia   25u  IPv6 60901679      0t0  TCP *:8444 (LISTEN)
chia_full  1687 chia   26u  IPv4 60901680      0t0  TCP *:8444 (LISTEN)
chia_full  1687 chia   30u  IPv4 60901694      0t0  TCP localhost:8444->localhost:48062 (ESTABLISHED)
chia_full  1687 chia   36u  IPv4 60901707      0t0  TCP localhost:8444->localhost:48078 (ESTABLISHED)
chia_full  1687 chia   59u  IPv4 60902003      0t0  TCP <myLocalHostname>:55984-><someRemoteIp>:8444 (CLOSE_WAIT)
chia_full  1690 chia   25u  IPv6 60901679      0t0  TCP *:8444 (LISTEN)
chia_full  1690 chia   26u  IPv4 60901680      0t0  TCP *:8444 (LISTEN)
```

Chia Log (level NOTICE) looks like this for inbound packets:
```
2021-06-09T21:44:05.881 full_node full_node_server        : INFO     Connection closed: <someRemoteIp>, node id: <someId>
2021-06-09T21:44:05.881 full_node chia.full_node.full_node: INFO     peer disconnected {'host': '<someRemoteIp>', 'port': 8444}
```

Am I alone with this issue? Any ideas or recommendations?

**Comment by joelrb** [comment]
You're not alone. I got tired of looking for solution of this issue. I only get 8 peers since joining last March. Played with turning off upnp, port forwarding, use another router, use another internet provider, connect another router to main router, use lan port only, use wifi, use cellphone hotspot, use another computer, clean install computer, etc etc... through all these I NEVER got over 8 peers... sigh...

**Comment by decea89** [comment]
I am on the same ship.... I've tried almost the same that you @joelrb. My ISP has changed my public IP several times lately when I've suffered light cuts in the building and my conclusion is that a lot of peers have banned me (I've seen the tcp sockets and the inbound connections via 8444 are in CLOSE_WAIT state, which means that they endpoint has closed the connection). I NEVER get more than 10-12 peers (only 3-4  inbound connections).. My next step is purchasing a static public IP in my ISP. On the other hand I am afraid of my node can be included in too many black lists. I am considering to farm through a new node when pools are open, since my solo plots are linked to my current node.
